setup.py

```diff
@@ -37,6 +37,7 @@ def get_advanced_templates():
         'aiohttp==0.22.5',
         'analytics-python',
         'coloredlogs',
+        'dnspython',
```

Comment:
I've removed it/ I am going to wait for the team-city to provide feedback.
Since `dns` module is being used by `test-upgrade-vpc` binary outside of `tox` run, I have a doubt that it will be available directly in the environment. If it is not, we have to do something.